scripts/update_mkdocs/templates/sample.md.jinja

```diff
@@ -0,0 +1,26 @@
+{#- mode: jinja -#}
+
+{% if samples %}
+### Raw Events Samples
+
+In this section, you will find examples of raw logs as generated natively by the source. These examples are provided to help integrators understand the data format before ingestion into Sekoia.io. It is crucial for setting up the correct parsing stages and ensuring that all relevant information is captured.
+
+{% for sample in samples %}
+=== "{{sample['name']}}"
+{% if is_sample_json %}
+
+    ```json
+	{% filter indent(width=4) %}
+{{sample['content']}}
+	{% endfilter %}
+	```
+{% else %}
+    ```
+	{% filter indent(width=4) %}
+{{sample['content']}}
+	{% endfilter %}
+	```
```

Comment:
I suggest a little improvement to avoid additional newlines and I fixed the indentation of backticks
```suggestion
    ```
	{% filter indent(width=4) %}{{sample['content']}}{% endfilter %}
    ```
```